### PR TITLE
Create audio settings

### PR DESCRIPTION
**Pull request by afichemist**

### CODE REVIEW

**Comment by GSculerlor** [comment]
Please just fix your PR instead of closing and create another PR everytime appveyor build is failed

**Comment by afichemist** [comment]
I Was See The CI Problem,but no whitespace can be trimmed.its separated by commented code.

File: osu.Game/Screens/Play/HUD/PlayerSettingsOverlay.cs [hunk 1]
```diff
@@ -13,18 +13,13 @@ namespace osu.Game.Screens.Play.HUD
...
     {
         private const int fade_duration = 200;
-
         public bool ReplayLoaded;
-
         public readonly PlaybackSettings PlaybackSettings;
-
         public readonly VisualSettings VisualSettings;
-
+        public readonly AudioSettings AudioSettings;
```
Comment (TPGPL) [inline osu.Game/Screens/Play/HUD/PlayerSettingsOverlay.cs]: space at the line ending

File: osu.Game/Screens/Play/HUD/PlayerSettingsOverlay.cs [hunk 2]
```diff
@@ -42,7 +37,8 @@ public PlayerSettingsOverlay()
                     //CollectionSettings = new CollectionSettings(),
                     //DiscussionSettings = new DiscussionSettings(),
                     PlaybackSettings = new PlaybackSettings(),
-                    VisualSettings = new VisualSettings { Expanded = false }
+                    VisualSettings = new VisualSettings { Expanded = false },
+                    AudioSettings = new AudioSettings{ Expanded = false }
```
Comment (bdach) [inline osu.Game/Screens/Play/HUD/PlayerSettingsOverlay.cs]:
```suggestion
                    AudioSettings = new AudioSettings { Expanded = false }
```

File: osu.Game/Screens/Play/PlayerSettings/VisualSettings.cs [hunk 3]
```diff
@@ -39,8 +38,7 @@ public VisualSettings()
                 },
                 showStoryboardToggle = new PlayerCheckbox { LabelText = "Storyboards" },
                 showVideoToggle = new PlayerCheckbox { LabelText = "Video" },
-                beatmapSkinsToggle = new PlayerCheckbox { LabelText = "Beatmap skins" },
-                beatmapHitsoundsToggle = new PlayerCheckbox { LabelText = "Beatmap hitsounds" }
+                beatmapSkinsToggle = new PlayerCheckbox { LabelText = "Beatmap skins"}
```
Comment (bdach) [inline osu.Game/Screens/Play/PlayerSettings/VisualSettings.cs]:
```suggestion
                beatmapSkinsToggle = new PlayerCheckbox { LabelText = "Beatmap skins" }
```

File: osu.Game/Screens/Play/HUD/PlayerSettingsOverlay.cs (hunk 1 above)
Comment (afichemist) [reply osu.Game/Screens/Play/HUD/PlayerSettingsOverlay.cs]: ok,thank for this. I wasn't see this space

File: osu.Game/Screens/Play/PlayerLoader.cs [hunk 4]
```diff
@@ -111,7 +111,7 @@ private void load(SessionStatics sessionStatics)
                     Children = new PlayerSettingsGroup[]
                     {
                         VisualSettings = new VisualSettings(),
-                        new InputSettings()
+                        new InputSettings(),new AudioSettings()
```
Comment (bdach) [inline osu.Game/Screens/Play/PlayerLoader.cs]:
```suggestion
                        new InputSettings(),
                        new AudioSettings()
```

**Review by bdach — CHANGES_REQUESTED** [review]

**Comment by bdach** [comment]
I don't mean to be rude, but:

* You don't seem to understand how `git` and pull requests work, despite me linking you some articles to read in #6618.
* You've closed #6621 despite my _explicit pleas_ to do so, giving an explanation that I do not understand.
* You've also ignored my comments about the other broken whitespace, which I pointed out explicitly above.

Let me be clear: as much as we might appreciate you wanting to chip in, you are creating way too much work for us to accept what should be a trivial change. If your PRs are going to require this much intervention going forward, it is likely they _will_ be dropped outright without explanation, as other contributors might not have as much patience as me.

File: osu.Game/Screens/Play/PlayerLoader.cs (hunk 4 above)
Comment (iiSaLMaN) [reply osu.Game/Screens/Play/PlayerLoader.cs]: Please do not mark conversations as "resolved" while they're actually not.

**Pull request closed without merging**